src/sage/combinat/matrices/hadamard_matrix.py

```diff
@@ -3279,6 +3371,8 @@ def symmetric_conference_matrix(n, check=True):
     - ``check`` -- boolean (default: ``True``); whether to check that output is
       correct before returning it. As this is expected to be useless, you may
       want to disable it whenever you want speed.
+    - ``existence`` -- boolean (default: ``False``); if true, only check that such
+      a matrix exists.
 
```

Comment:
```sage
a matrix exists
```
no period at the end for one sentence description.

Reply:
Sorry, note that there is a period in the previous item `check` as well - and this is how it's in most of Sage code I ever touched, IMHO.

Reply:
the style of this file is that if there is a period at the end of an item, then it keeps so for the rest of the list.

Reply:
I don't agree. But okay as it is trivial.